### ISSUE TITLE
upload documents while creating task

### ISSUE THREAD

**Issue by skywalker89** [kanboard/forum#12]
HI, right now I cannot add a document while creating the task... I have to save and reopen to find "upload"

Would be great if I could create the task 100% in one sweep... but no big deal, just an idea

Thx for your work!

Dan

**Comment by kyrgorsh** [comment]
Yeah! Great someone also needs this feature. I do agree with the author, would be nice if there is a pssibility to attach files during task creation
